ocs_ci/utility/rosa.py

```diff
@@ -596,3 +598,57 @@ def is_odf_addon_installed(cluster_name=None):
         return True
     else:
         return False
+
+
+def post_onboarding_verification():
+    """
+    Check that after onboarding consumer the relevant resources
+    were created in the provider cluster and then switch back to the original cluster:
+    1. StorageConsumer with the correct id exists and all its CephResources
+    are in Ready status
+    2. CephBlockPool and Subvolumegroup with the correct id are in Ready status
+    """
+    logger.info("Starting post-onboarding verification")
+    if config.multicluster:
```

Comment:
There could be a log message if multicluster is not set that no post-onboarding validation is done.